### PR TITLE
Added Article about Operators In Python

### PR DESCRIPTION
**Pull request by T0MGithub**
This article doesn't include all the operators in Python, but covers the most common operators - arithmetic, comparison and assignment operators as well as operator precedence, explaining the order they are executed in. This article is more geared towards beginners due to only covering common, mostly simple operators, but if you guys think it is a useful article, could be expanded in future to cover other operators such as bitwise operators. However that article may reach several hundred lines and may need to be split up into different articles. Make any edits to improve the explanations or examples you want.

### CODE REVIEW

File: src/content/resources/python/operators.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+```
+
+#### Lesser Than Operators - `<`
+
+Checks whether the left hand operand is numerically smaller than the right hand operand, returning `True` if it is and `False` otherwise. This operand can compare other data types, but always returns `False`.
+
+```python
+7 < 9 # returns True
+8 < 8 # returns False
+9 < 9 # retursn False
```
Comment (veksen) [inline src/content/resources/python/operators.md]: typo

File: src/content/resources/python/operators.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+## Python Operators
+
+Python has 7 categories of operators: Arithmetic, assignment, comparison, logical, identity, membership and bitwise operators. In this article I will cover some of the operators seen most commonly in Python programming - arithmetic, assignment and comparison. Note: this article covers Python 3 and there have been alterations and additions of operators since Python 2, so some elements of this article may only apply to Python 3.
+
+## Arithmetic Operators
+
+Arithmetic operators are used in conjunction with numeric values in order to perform basic mathematical operations.
+
+#### Addition Operator - `+`
```
Comment (veksen) [inline src/content/resources/python/operators.md]: Can subheadings be h3 instead of h4? `###`

**Comment by veksen** [comment]
Looks good to me, left some comments. If you remember, for the next PR, if you can create a new branch, and not merge from your own master, it would be better!

Thank you very much!

**Comment by T0MGithub** [comment]
I messed up one the commit messages (says h2 instead h4). Sorry about the branches, should have though about it.

**Comment by T0MGithub** [comment]
Anything else you need me to change?

**Comment by veksen** [comment]
No I've been very busy, merging this!

**Pull request merged**